src/components/ProductCard/ProductCard.jsx

```diff
@@ -2,19 +2,22 @@ import * as React from "react"
 import "./ProductCard.css"
 
 export default function Products(props) {
-  const Operations = Object.freeze({
-    Add: Symbol("add"),
-    Subtract: Symbol("subtract"),
-    Erase: Symbol("erase")
-  })
-
+  /*
+  operations enum format:
+    const operations = Object.freeze({
+      Add: Symbol("add"),
+      Subtract: Symbol("subtract"),
+      Erase: Symbol("erase")
+    })
+  */
+  
```

Comment:
You can delete this if you don't need it. You can always look at the code history in git/github if you need to find old code.